test/recipes/30-test_evp.t

```diff
@@ -133,9 +134,14 @@ push @defltfiles, qw(evpciph_aes_gcm_siv.txt) unless $no_siv;
 push @defltfiles, qw(evpciph_aes_siv.txt) unless $no_siv;
 push @defltfiles, qw(evpkdf_argon2.txt) unless $no_argon2;
 
+# A list of tests that only run with the FIPS provider
+# (i.e. The algorithms are not suitable with the default provider)
+my @fipsfiles = qw(fips_mac.txt);
```

Comment:
Why do we need this list? We have 

Availablein=fips

That can be used in .txt files already.

Reply:
Yeah, me forgetting :(

Reply:
Still, I'm unsure if a separate FIPS selection isn't a good idea longer term.

It isolates the insanity to a few easily distinguished files.

Reply:
That would be a separate effort if we decide to do that at some point..